packages/api/src/external/carequality/document/query-documents.ts

```diff
@@ -23,11 +26,19 @@ export async function getDocumentsFromCQ({
   requestId: string;
   patient: Patient;
 }) {
-  if (!iheGateway) return;
+  const { log } = out(`CQ DQ - requestId ${requestId}, M patient ${patient.id}`);
+  const { cxId, id: patientId } = patient;
+
+  const interrupt = buildInterrupt({ patientId, cxId, log });
+  if (!iheGateway) return interrupt(`IHE GW not available`);
+  if (!lambdaName) return interrupt(`IHE DR lambda not available`);
+  if (!(await isCQDirectEnabledForCx(cxId))) return interrupt(`CQ disabled for cx ${cxId}`);
```

Comment:
If the IHE GW is not available or CQ not enabled for this cx, also reset the CQ doc query progress for this patient.